### PR TITLE
Negate some ifs to unindent the code

### PR DESCRIPTION
**Pull request by rluvaton**
This PR fixes nothing

### CODE REVIEW

**Comment by rluvaton** [comment]
It seems like that For some reason Firefox (for playwright) failed to install on MacOS because some disconnection problem, If someone can rerun the tests it would be great

**Comment by hediet** [comment]
Thanks for your PR! I appreciate your effort. While I agree that the nesting should be reduced somehow in this code, I don't think your change makes the code more readable.
*Early return* usually indicates a happy path. In this case, the early returns you added would indicate an happy path in which both render data is set and line count > 0 is true. I'm not sure that this path is obviously a happy one and that it is obvious to a reader what an early return skips, especially as there are side effects.

**Comment by rluvaton** [comment]
Thank you @hediet ! for your deep explanation.

**Comment by hediet** [comment]
I think you should only return early if you know immediately why returning early is ok, without reading the code after the return statement.
In this case I don't find it obvious why returning is the right thing to do without looking at the code lines below the return.

File: src/vs/editor/contrib/inlineCompletions/ghostTextWidget.ts [hunk 1]
```diff
@@ -216,24 +216,30 @@ export class GhostTextWidget extends Disposable {
...
-							heightInLines: heightInLines,
-							domNode,
-						});
-					} else if (remainingLines.length > 0) {
-						this.viewMoreContentWidget = this.renderViewMoreLines(renderData.position, renderData.lines[0], remainingLines.length);
-					}
-				}
+			if (!renderData) {
+				return;
+			}
```
Comment (rluvaton) [inline src/vs/editor/contrib/inlineCompletions/ghostTextWidget.ts:221]: So I think that @hediet is good

**Pull request closed without merging**